plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/edit.scss

```diff
@@ -1,4 +1,24 @@
-.wc-block-editor-add-to-cart-with-options__skeleton-wrapper {
-	max-width: 300px;
-	margin-bottom: $gap;
+.wp-block-woocommerce-add-to-cart-with-options {
+	&__skeleton-wrapper {
```

Comment:
Personal opinion here, but I usually prefer to keep class names complete in SCSS files so they are easily searchable. Ie, right now if I search for `wp-block-woocommerce-add-to-cart-with-options__skeleton-wrapper` in the codebase, I wouldn't be able to find this selector and I could mistakenly think the class is not used.

Reply:
done -> [2065579](https://github.com/woocommerce/woocommerce/pull/55610/commits/20655799855610e0c5cd668b3aef198c4de15b0c)